scripts/release/create-release.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+echo "DEPRECATED!!! USE create-release.hs INSTEAD!!!"
```

Comment:
Nice. Although can you remind me why we did not remove this script altogether?

Reply:
no particular reason, I'm more than happy to remove it in this PR instead of just adding a warning